src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -648,7 +648,7 @@ internal sealed override bool GenerateDebugInfo
 
                 if (IsAsync)
                 {
-                    // PROTOTYPE: Need more dedicated debug information testing when runtime async is enabled.
+                    // https://github.com/dotnet/roslyn/issues/79792: Need more dedicated debug information testing when runtime async is enabled.
```

Comment:
This links to the "Use MethodImplAttributes.Async" issue. Is that the right issue?